ironfish/src/network/peers/peerConnectionManager.ts

```diff
@@ -110,21 +110,11 @@ export class PeerConnectionManager {
   }
 
   private connectToEligiblePeers(peer: Peer): boolean {
-    if (peer.state.type !== 'CONNECTED') {
-      if (this.peerManager.canConnectToWebRTC(peer)) {
-        if (this.peerManager.connectToWebRTC(peer)) {
-          return true
-        }
-      }
-
-      if (this.peerManager.canConnectToWebSocket(peer)) {
```

Comment:
canConnectToWebSocket is called in connectToWebSocket

Reply:
<img width="897" alt="image" src="https://github.com/iron-fish/ironfish/assets/13268167/dd0937cd-9d56-421d-b234-94a24051b6e6">